include/vkgcDefs.h

```diff
@@ -47,7 +47,7 @@
 #define LLPC_INTERFACE_MAJOR_VERSION 45
 
 /// LLPC minor interface version.
-#define LLPC_INTERFACE_MINOR_VERSION 2
+#define LLPC_INTERFACE_MINOR_VERSION 3
```

Comment:
This PR (https://github.com/GPUOpen-Drivers/llpc/pull/1094) also revised the same minor version. Pay attention to the PR merging order. If conflicted, please rebase and increment this value.

Reply:
I suppose there will be conflicts after merge #1094, but now it is merge, this PR doesn't show conflicts...

Reply:
I guess git sees that it will be changed to the state that we already have in `dev`, so from git’s perspective, it is fine to be merged as the result is identical.
If Tim set it to 4 from the start, git would have detected a conflicting change.

Reply:
got it

Reply:
For clarification, as far as I understand @amdrexu’s comment, @tcorringham should now rebase and increment the minor version to 4.

Reply:
yeah, I can do it in a following commit, no need to bother him